src/main/steps/applicant1/review-your-joint-application/content.ts

```diff
@@ -11,6 +11,7 @@ import { enDomicile, enHabitualResident } from '../../../app/jurisdiction/moreDe
 import { CommonContent } from '../../common/common.content';
 import { accessibleDetailsSpan, formattedCaseId, getAppSolAddressFields } from '../../common/content.utils';
 import { CHECK_CONTACT_DETAILS } from '../../urls';
+import { getName } from '../hub-page/content';
```

Comment:
could getName be put somewhere more generic?

Reply:
moved it to content.utils file and added test. also cleaned up the function abit to remove undefined values.